services/bulletin/src/main/java/edu/uga/devdogs/bulletin/webscraping/PrerequisiteGroup.java

```diff
@@ -0,0 +1,74 @@
+package edu.uga.devdogs.bulletin.webscraping;
+
+import java.util.ArrayList;
+
+/**
+ * @author Whitman Stewart
+ * A prerequisite group represents any of the conditions that would meet  one part of a course's requirements.
+ * For example, CSCI 4730's requirements -- (CSCI 4720 or CSCI 4720E or CSEE 4280) and CSCI 2720 -- would have two
+ * prerequisite groups: "CSCI 4720 or CSCI 4720E or CSEE 4280", and "CSCI 2720."
+ */
+public class PrerequisiteGroup {
+    private boolean requiresHonors;
```

Comment:
to stay consistent throughout the project please group all class variables together here at the top instead of interweaving them throughout the code.